sdk/core/Azure.Core.TestFramework/src/RecordMatcher.cs

```diff
@@ -13,13 +13,9 @@ namespace Azure.Core.TestFramework
 {
     public class RecordMatcher
     {
-        // Headers that are normalized by HttpClient
-        private HashSet<string> _normalizedHeaders = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
-        {
-            "Accept",
-            "Content-Type"
-        };
-
+        /// <summary>
```

Comment:
Why keep this type?

Reply:
I liked having the conceptual separation of sanitization and matching. I guess this can be renamed to SanitizationOptions and MatchingOptions and then make the properties not be settable.

And welcome @pakrym!

Reply:
Looking at some of the usages, there are cases where libraries (e.g. Communication) have CustomSanitizerB deriving from CustomSantizerA, but there is no corresponding inheritance hierarchy for CustomLiveTestB/CustomLiveTestA. So allowing users to create these sanitizers and derive from them is helpful to avoid duplicating the same settings in multiple places when they don't intend to user inheritance in the live test classes. I still think it makes sense to at least rename these types to be suffixed with Options.

Reply:
Up to you but that inheritance hierarchy in communication is not worth having. The base type has a single header in it.

Reply:
Fair point - I will defer to your judgement here!